superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterControl.tsx

```diff
@@ -219,6 +221,14 @@ export default function DateFilterControl(props: DateFilterLabelProps) {
     });
   }, [value]);
 
+  useEffect(() => {
+    if (isMounted) {
+      onChange('Last week');
+      setTimeRangeValue('Last week');
+      setFrame(guessFrame('Last week'));
```

Comment:
These two lines can be removed. Because when the `value` changed(onChange('last week')) then the component can reload.

Reply:
Sorry. Ignore me. Shouldn't be removed. merged.